.changeset/neat-cooks-type.md

```diff
@@ -0,0 +1,41 @@
+---
+'@shopify/shopify-app-session-storage-prisma': minor
+'@shopify/shopify-app-session-storage-test-utils': patch
```

Comment:
Are these minor and patch changes?

Both add new features, so that seems like it's at-least a patch.  If the user updates the dependency without updating the schema then does it break? In which case, it's a major?

I think at minimum these should be patch, and we should include the steps the user needs to take to update their schema?  But I also want to make sure that if these are actually major, they are marked as major.